src/train_gui.cpp

```diff
@@ -322,19 +322,14 @@ int GetTrainDetailsWndVScroll(VehicleID veh_id, TrainDetailsWindowTabs det_tab)
 	int num = 0;
 
 	if (det_tab == TDW_TAB_TOTALS) { // Total cargo tab
-		CargoArray act_cargo;
-		CargoArray max_cargo;
+		CargoArray act_cargo{};
```

Comment:
Since we're only counting what's in `max_cargo`, is `act_cargo` actually needed? Can't it be removed completely?

Reply:
Yeah I didn't really look at what it was doing with it :)